backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/references/RelationTypeZoomProvider.java

```diff
@@ -309,6 +310,11 @@ private String createZoomOriginQueryWhereClause(final IZoomSource zoomOrigin)
 						.append(originTableName)
 						.append(".")
 						.append(originRecordIdName);
+				if (whereClause != null)
```

Comment:
```suggestion
				if (Check.isNotBlank(whereClause))
```

...with `de.metas.util.Check`...this also takes care of e.g. `"    "`